tests/test-sources/plugins/utils/auto-save.nix

```diff
@@ -3,47 +3,71 @@
     plugins.auto-save.enable = true;
   };
 
-  defaults = {
+  example = {
     plugins.auto-save = {
       enable = true;
 
-      keymaps = {
-        silent = true;
-        toggle = "<leader>s";
-      };
-      enableAutoSave = true;
-      executionMessage = {
-        message.__raw = ''
-          function()
-            return ("AutoSave: saved at " .. vim.fn.strftime("%H:%M:%S"))
+      settings = {
+        condition = ''
+          function(buf)
+            local fn = vim.fn
+            local utils = require("auto-save.utils.data")
+
+            if utils.not_in(fn.getbufvar(buf, "&filetype"), {'oil'}) then
+              return true
+            end
+            return false
           end
         '';
-        dim = 0.18;
-        cleaningInterval = 1250;
+        write_all_buffers = true;
+        debounce_delay = 1000;
       };
-      triggerEvents = [
-        "InsertLeave"
-        "TextChanged"
-      ];
-      condition = ''
-        function(buf)
-          local fn = vim.fn
-          local utils = require("auto-save.utils.data")
+    };
+  };
 
-          if fn.getbufvar(buf, "&modifiable") == 1 and utils.not_in(fn.getbufvar(buf, "&filetype"), {}) then
-            return true -- met condition(s), can save
+  defaults = {
+    plugins.auto-save = {
+      enable = true;
+
+      settings = {
+        enabled = true;
+        executionMessage = {
```

Comment:
Settings names use the same casing as upstream (i.e. snake_case here: `execution_message`)